### PR TITLE
Upgrading react-router.

### PR DESCRIPTION
**Pull request by mkucharz**
Resolving #15

### CODE REVIEW

**Comment by kkkowalski** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "classnames": "^2.2.1",
     "d.js": "^0.7.3",
+    "flexboxgrid": "^6.3.0",
```
Comment (kkotelczuk) [inline package.json:14]: Is that package used by router?

Comment (mkucharz) [inline package.json:14]: No, I added it in meantime because for some reason I had a problem with this dependency

Comment (kkotelczuk) [inline package.json:14]: Ok

**Pull request merged**